site/content/community/guides.en.md

```diff
@@ -2,25 +2,26 @@ Share your applications, videos, and blog posts with fellow Copilots!
-| [**AWS Copilot: an application-first CLI for containers on AWS**](https://aws.amazon.com/blogs/containers/aws-copilot-an-application-first-cli-for-containers-on-aws/) by <a href="https://twitter.com/efekarakus">@efekarakus</a> | Efe walks through the design tenets of the CLI: why they were chosen, how they map to Copilot features, and the vision for how the CLI will evolve in the future.  |
-| [**Introducing AWS Copilot**](https://aws.amazon.com/blogs/containers/introducing-aws-copilot/) by <a href="https://twitter.com/nathankpeck">@nathankpeck</a> | Nathan explains how with the AWS Copilot CLI you can go from idea to implementation much faster, with the confidence that the infrastructure you have deployed has production-ready configuration. |
+| Title                                                                                                                                                                                                                                                                                            | Description                                                                                                                                                                                                                                                                                                                                                            |
+|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [**Prefect Cloud based data pipelines using AWS Copilot on ECS**](https://www.core27.co/post/prefect-cloud-based-data-pipelines-using-aws-copilot-on-ecs) by <a href="https://www.reddit.com/r/dataengineering/">@dataengineering </a>                                                                                 | Data Engineering takes you through build and deploy a production grade data stack on Amazon's Elastic Container Service.                                                                                                                                                                                                             |
```

Comment:
The blog was posted on Reddit by https://www.reddit.com/user/_ragequilt_/ but for the author we should do:
```suggestion
| [**Prefect Cloud based data pipelines using AWS Copilot on ECS**](https://www.core27.co/post/prefect-cloud-based-data-pipelines-using-aws-copilot-on-ecs) by <a href="https://twitter.com/Core27_">@Core27_</a>                                                                                 | Data Engineering takes you through build and deploy a production grade data stack on Amazon's Elastic Container Service.                                                                                                                                                                                                             |
```